core/src/main/scala/kafka/controller/ControllerContext.scala

```diff
@@ -126,6 +130,26 @@ class ControllerContext {
     replicaStates.clear()
   }
 
+  def addTopicId(topic: String, id: UUID): Unit = {
+    if (!allTopics.contains(topic))
+      throw new IllegalStateException("topic " + topic +  " is not contained in all topics.")
```

Comment:
We can use string interpolation with these to make them a little bit neater e.g. `s"topic $topic is not contained in all topics"`.